#include <iostream>
using namespace std;
int main()
{
  int a,b,x;
  cin>>a>>b;
  try
  {
    if(b==0)
    {
       throw(x);
    }
    cout<<"VALID";
  }
catch(int y)
{
  cout<<"EXCEPTION Not possible";
}
	return 0;
}